config.md

```diff
@@ -308,7 +308,10 @@ The semantics are the same as `Path`, `Args` and `Env` in [golang Cmd](https://g
 
 This OPTIONAL property contains arbitrary metadata for the container.
 This information MAY be structured or unstructured.
-Annotations are key-value maps.
+Annotations MUST be key-value maps.
+Keys MUST be unique, and best practice is to namespace the keys.
```

Comment:
On Tue, Jun 28, 2016 at 02:16:22PM -0700, Mrunal Patel wrote:
> > +Keys MUST be unique, and best practice is to namespace the keys.
> 
> How about "acme.com/gpu-cores" ? :)

example.com/gpu-cores?  acme.com is a real thing:

  $ whois acme.com | grep -A1 'Registrant Name'
  Registrant Name: JEFFREY POSKANZER
  Registrant Organization: ACME LABORATORIES